recipe/meta.yaml

```
@@ -29,21 +29,28 @@ outputs:
 
     requirements:
       host:
-        - python >=3.7
+        - python
         - pip
+        - wheel
+        - setuptools
       run:
-        - python >=3.7
-        - numpy >=1.9.0
+        - python
+        - numpy >=1.19.0
         - pandas >=1.0
-        - typing_extensions
+        - typing_extensions >=3.7.4.3
```

```suggestion
        - typing_extensions >=3.7.4.3  # [py<38]  
```
https://github.com/unionai-oss/pandera/blob/v0.20.4/setup.py#L55